src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,78 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String coma = ",";
+    public static final String supply = "supply";
+    public static final String buy = "buy";
+    public static final String result = "result";
+    public static final int index0 = 0;
+    public static final int index1 = 1;
```

Comment:
it's a bad practice to use number in variable/constants names. Rename them to `OPERATION_INDEX `, `AMOUNT_INDEX`